core/templates/dev/head/services/PlaythroughIssuesBackendApiService.js

```diff
@@ -19,10 +19,6 @@
 oppia.constant(
   'FETCH_ISSUES_URL', '/issuesdatahandler/<exploration_id>');
```

Comment:
Maybe off topic here, but does this handler only cater to playthrough issues or all issues?